lib/utils/loadImages.js

```diff
@@ -0,0 +1,18 @@
+function createImageList () {
```

Comment:
This is pretty opaque, what are you doing here? Could you add a comment?

Reply:
done